test/tests/functional/pbs_reservations.py

```diff
@@ -1074,22 +1070,18 @@ def test_advance_resv_with_multinode_job_array(self):
                            extend='t', id=jid2)
         self.server.expect(JOB, {'job_state=Q': 3}, count=True,
                            extend='t', id=jid2)
-        # Suspend first and second sub-jobs
         self.server.sigjob(jobid=subjid2[0], signal="suspend")
         self.server.expect(JOB, {'job_state': 'S'}, id=subjid2[0])
         self.server.sigjob(jobid=subjid2[1], signal="suspend")
         self.server.expect(JOB, {'job_state': 'S'}, id=subjid2[1])
-        # Verify third and fourth subjobs are running
         self.server.expect(JOB, {'job_state': 'R'}, id=subjid2[2])
         self.server.expect(JOB, {'job_state': 'R'}, id=subjid2[3])
-        # Delete running jobs
-        self.server.delete(subjid2[2], subjid2[3])
-        # Verify fifth subjob is in R state
+        self.server.delete([subjid2[2], subjid2[3]])
         self.server.expect(JOB, {'job_state': 'R'}, id=subjid2[4])
-        # Resume the suspended jobs
+        self.server.expect(JOB, {'job_state': 'X'}, id=subjid2[4], offset=60)
         self.server.sigjob(subjid2[0], 'resume')
         self.server.sigjob(subjid2[1], 'resume')
         self.server.expect(JOB, {'job_state=R': 2}, count=True,
                            extend='t', id=jid2)
         self.server.expect(JOB, {'job_state': 'F'},
-                           id=jid, extend='x', offset=180)
+                           id=jid2, extend='x', offset=180)
```

Comment:
Consider adding a self.logger.info() explaining what the 120s sleep is for.  It'd be nicer to the test runner to know why they are waiting.  It isn't necessary though.